src/inet/tests/BUILD.gn

```diff
@@ -37,8 +37,6 @@ chip_test_suite("tests") {
     "TestInetLayer.cpp",
     "TestInetLayer.h",
     "TestInetLayerCommon.cpp",
-    "TestInetLayerDNS.cpp",
```

Comment:
why are these removed?

Reply:
This is not removed, but was moved to line 67. Also this is a merge commit, the original change was made on master in other prs.